apps/papra-server/src/modules/intake-emails/username-drivers/pattern/pattern.intake-email-username-driver.constants.ts

```diff
@@ -0,0 +1,8 @@
+export const PATTERNS_PLACEHOLDERS = {
```

Comment:
An explanation would be great. USER_NAME and USER_EMAIL_USERNAME are pretty identical